libs/usb-drive/src/mocks/memory_usb_drive.ts

```diff
@@ -18,18 +18,6 @@ export interface MockUsbDrive {
   removeUsbDrive(): void;
 }
 
-// Clean up any mock USB drives directories that were created during tests
-if (typeof jest !== 'undefined') {
```

Comment:
We can't (afaik) cleanly conditionally run code in the vitest environment, so I replaced this with an assumption that the `@votingworks/fixtures` cleanup function would be running for any test that uses this.